### PR TITLE
[7.x] Fix tests

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

**Comment by driesvints** [comment]
Ping @crynobone

File: src/Console/DuskCommand.php [hunk 1]
```diff
@@ -282,7 +283,11 @@ protected function writeConfiguration()
     {
         if (! file_exists($file = base_path('phpunit.dusk.xml')) &&
             ! file_exists(base_path('phpunit.dusk.xml.dist'))) {
-            copy(realpath(__DIR__.'/../../stubs/phpunit.xml'), $file);
+            if (version_compare(Application::VERSION, '10.0.0') >= 0) {
```
Comment (crynobone) [inline src/Console/DuskCommand.php:286]:
```suggestion
            if (version_compare(Application::VERSION, '10.0', '>=')) {
```

Comment (crynobone) [reply src/Console/DuskCommand.php:286]: But this doesn't guarantee PHPUnit 10 is installed, maybe use `version_compare()` against phpunit instead?

**Comment by taylorotwell** [comment]
@driesvints please mark as ready for review when feedback addressed.

File: src/Console/DuskCommand.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 use Dotenv\Dotenv;
 use Illuminate\Console\Command;
+use Illuminate\Foundation\Application;
 use Illuminate\Support\Str;
```
Comment (crynobone) [inline src/Console/DuskCommand.php:8]:
```suggestion
use Illuminate\Support\Str;
use PHPUnit\Runner\Version;
```

File: src/Console/DuskCommand.php (hunk 1 above)
Comment (crynobone) [inline src/Console/DuskCommand.php:286]:
```suggestion
            if (version_compare(Version::id(), '10.0', '>=')) {
```

**Comment by driesvints** [comment]
Ready again.

**Pull request merged**